.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "8.10"
```

Comment:
We may want to tweak this based on what we decide are minimum node/npm versions needed